src/main/java/frc/robot/scoreStates/ScoreStateConstants.java

```diff
@@ -22,10 +22,10 @@ public class ScoreStateConstants {
                     .map(GeometryUtil::flipFieldPosition)
                     .collect(Collectors.toList());
 
-    public static final List<Pose2d> OPTIMAL_POINTS_TRAP_BLUE =
+    public static final List<Pose2d> OPTIMAL_POINTS_CLIMB_BLUE =
             List.of(new Pose2d(0, 0, Rotation2d.fromDegrees(0)));
-    public static final List<Pose2d> OPTIMAL_POINTS_TRAP_RED =
-            OPTIMAL_POINTS_TRAP_BLUE.stream()
+    public static final List<Pose2d> OPTIMAL_POINTS_CLIMB_RED =
+            OPTIMAL_POINTS_CLIMB_BLUE.stream()
                     .map(GeometryUtil::flipFieldPose)
                     .collect(Collectors.toList());
     public static final Translation2d AMP_POSE_BLUE = new Translation2d(1.85, 7.49);
```

Comment:
You might want to keep `AMP_POSE_RED` and `AMP_POSE_BLUE` seperated to ease tuning on the field in the competition -- there is some tolerance there. However, it's probably not significant enough, so choose what you think is better.

Reply:
I'll take your suggestion
49cb6e45a8dec6b90c30e40c3af4553d8d4a3994

Reply:
🤗
I would though write already the position of the Amp from the manual or CAD.

Reply:
ead405e